docs/06-concepts/11-authentication/01-setup.md

```diff
@@ -187,20 +184,64 @@ void main() async {
 
 The `SessionManager` has useful methods for viewing and monitoring the user's current state:
 
-- The `signedInUser` will return a `UserInfo` if the user is currently signed in (or `null` if the user isn't signed in).
-- Use the `addListener` method to get notified of changes to the user's signed in state.
-- Sign out a user by calling the `signOut` method.
-
-For example it can be useful to subscribe to changes in the `SessionManager` and force a rerender of your app.
-
+- **Check authentication state**  
```

Comment:
Suggestion: The descriptions here do not match in style. Make sure it is a common language across all these.